bin/run-node-tests.js

```diff
@@ -22,8 +22,10 @@ if ('EMBER_CLI_TEST_OUTPUT' in process.env) {
 
 // Executes a command and pipes stdout back to the user.
 function exec(command, args) {
-  execa.sync(command, args, {
-    stdio: 'inherit',
-    preferLocal: true
-  });
+  try {
+    execa.sync(command, args, {
+      stdio: 'inherit',
+      preferLocal: true
+    });
+  } catch (e) { }
```

Comment:
Should the error at least be logged?

Reply:
execa throws an exception if the return value is non-zero, so you end up with a useless stack trace every time the tests fail. Since we pipe stderr to the parent stderr you will see the error messages anyway. You're right though that we should probably check to verify that the error isn't caused by a bad path or something and is really a non-zero exit code.

Reply:
Thanks for the context. Even if nothing is handled, a comment would probably help in regards to why the error is ignored 👍

Reply:
@tomdale - Can you confirm that failing node tests properly fail the build now?  My concern is that swallowing this error, will "eat" the non-zero exit code and the tests would seen as "passing" (due to process exiting with 0 exit code) when they actually failed...

Reply:
Would like to not repeat what is done in the VM when it comes to swallowing the error.

Reply:
haha, indeed!

Reply:
@knownasilya The release version of execa hides all information from you in the case of a non-zero exit code in the sync case, which is really unfortunate. It looks like a PR was accepted 6 days ago to address this but hasn't been released yet.

@rwjblue In my tests, running `yarn test` with a failing Node test produces an exit code of `1`, so I think we're good. I would expect Testem to treat TAP output that contained `not ok` as a failure, which it seems to be doing.